db/migrate/20241104170456_add_student_loan_interest_ded_amount_to_state_file_md_intakes.rb

```diff
@@ -0,0 +1,6 @@
+class AddStudentLoanInterestDedAmountToStateFileMdIntakes < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_md_intakes, :student_loan_interest_ded_amount_cents, :bigint, default: 0, null: false
```

Comment:
can we call this `primary_student_loan_interest_ded_amount`
also `spouse_student_loan_interest_ded_amount` (remove `_cents` for both)

can we also make this a decimal column like all our other `_amount` or `_amt` fields?

ex:

```
    add_column :state_file_az_intakes, :charitable_cash_amount, :decimal, precision: 12, scale: 2
```

Reply:
updated this column, and added a feature spec context for mfj to include this added two income sub form 👍